.buildkite/x-pack/pipeline.xpack.libbeat.yml

```diff
@@ -18,45 +14,176 @@ env:
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Ubuntu Unit Tests"
+
+      - label: ":go: Go Integration Tests"
+        key: "mandatory-int-test"
+        command: |
+          set -euo pipefail
+          cd x-pack/libbeat
+          mage goIntegTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Go Integration Tests"
+
+      - label: ":python: Python Integration Tests"
+        key: "mandatory-python-int-test"
+        command: |
+          set -euo pipefail
+          cd x-pack/libbeat
+          mage pythonIntegTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Python Integration Tests"
+
+      - label: ":windows: Windows 2016 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/libbeat
+          mage -w reader\etw build goUnitTest
+        key: "mandatory-win-2016-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2016}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Windows 2016 Unit Tests"
+
+      - label: ":windows: Windows 2022 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/libbeat
+          mage -w reader\etw build goUnitTest
+        key: "mandatory-win-2022-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2022}"
+          machine_type: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/libbeat/build/*.xml"
+          - "x-pack/libbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/libbeat: Windows 2022 Unit Tests"
+
+  ### TODO: this condition will be changed in the Phase 3 of the Migration Plan https://docs.google.com/document/d/1IPNprVtcnHlem-uyGZM0zGzhfUuFAh4LeSl9JFHMSZQ/edit#heading=h.sltz78yy249h
+  - group: "Extended Windows Tests"
+    key: "extended-win-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false"
```

Comment:
see https://github.com/elastic/beats/pull/38828/files#diff-4291f1a0414f4870861926dce4a866be0c14ce2b86f018e958458c53c719c7a2R104 should be `if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*[Ww]indows.*/`